app/controllers/merchant_items_controller.rb

```diff
@@ -1,6 +1,9 @@
 class MerchantItemsController < ApplicationController
 
   def show
-    
+    # require "pry"; binding.pry
```

Comment:
Can you remove this pry? Trying to keep commented code off main so we don't have too much to find later

Reply:
hell yeah good catch